pkg/db/db.go

```diff
@@ -247,3 +250,18 @@ func deserializeRecord(r *bufio.Reader) (key string, val []byte, seq uint64, err
 	}
 	return
 }
+
+func packSeq(seq uint64, repro bool) uint64 {
+	if repro {
+		seq |= (uint64(1) << 63)
+	}
+	return seq
+}
+
+func getSeq(seq uint64) uint64 {
```

Comment:
`unpackSeq(seq uint64) (uint64, bool)` would be more symmetric to `packSeq`.
Go has multiple return values, no reason to not use that.